src/components/BotBuilder/BotBuilder.js

```diff
@@ -58,45 +59,96 @@ class BotBuilder extends React.Component {
     }
   };
 
-  showChatbots = bots => {
+  getBotImage = bots => {
     let chatbots = [];
-    this.setState({
-      chatbots: [],
-    });
     if (bots) {
       bots.forEach(bot => {
-        chatbots.push(
-          <Card key={bot.name} className="bot-template-card">
-            <Link
-              to={
-                '/botbuilder/botwizard?name=' +
-                bot.name +
-                '&language=' +
-                bot.language +
-                '&group=' +
-                bot.group
-              }
-            >
-              <RaisedButton
-                label={bot.name}
-                labelPosition="before"
-                labelStyle={{ verticalAlign: 'middle' }}
-                backgroundColor={colors.header}
-                labelColor="#fff"
-              />
-            </Link>
-            <div className="bot-delete">
-              <Delete
-                color="rgb(255, 255, 255)"
-                onClick={() =>
-                  this.deleteBot(bot.name, bot.language, bot.group)
-                }
-              />
-            </div>
-          </Card>,
-        );
+        let name = bot.name;
+        let language = bot.language;
+        let group = bot.group;
+        let url =
+          BASE_URL +
+          '/cms/getSkill.json?group=' +
+          group +
+          '&language=' +
+          language +
+          '&skill=' +
+          name +
+          '&private=1&access_token=' +
+          cookies.get('loggedIn');
+        $.ajax({
+          url: url,
+          dataType: 'jsonp',
+          crossDomain: true,
+          success: function(data) {
+            data = data.text.split('\n::');
+            bot.image = data[23].split(' ')[1];
```

Comment:
Please use instead `data.text.match(/^::image\s(.*)$/m)` to extract the image name. Its not necessary that the image will be at the `23` index always.

Reply:
@AdityaSrivast Please use the regex pattern as suggested by @hkedia321 .